frontend/.env.development

```diff
@@ -1,4 +1,5 @@
-NEXT_PUBLIC_ALGOLIA_APP_ID=XQEP3AD9ZT
+# NEXT_PUBLIC_ALGOLIA_APP_ID=XQEP3AD9ZT
+NEXT_PUBLIC_ALGOLIA_APP_ID=9I2C2MSPT7
```

Comment:
As mentioned in the PR description, there's a deploy block while we wait for the Algolia support ticket to go through

> ## Notes
>
>:warning: currently this branch is using an Algolia app from my personal account, because we're waiting for Algolia to enable the "Facet display" feature on iFixit account.

Reply:
> As mentioned in the PR description, there's a deploy block while we wait for the Algolia support ticket to go through

Sorry I didn't see that :-(

Is this something that this pull should be blocked on? If we switch to the old account, does this pull break? Or is the experience jut not optimal?